syntax = "proto3";

package auth;

option go_package = "./auth";

service Auth {
  rpc CheckUserExisting(CheckUserExistingRequest) returns (CheckUserExistingResponse) {}
}


// CheckUserExisting()

message CheckUserExistingRequest {
  string user_id = 1;
}

message CheckUserExistingResponse {
  string id = 1;
  string email = 2;
  string first_name = 3;
  string last_name = 4;
  bool is_superuser = 5;
  bool is_active = 6;
  string created_at = 7;
  string updated_at = 8;
}
